python/ml4ir/applications/ranking/tests/data/configs/feature_config.yaml

```diff
@@ -62,6 +62,15 @@ features:
     feature_layer_info:
       type: numeric
       shape: null
+      fn: tf_native_op
+      args:
+        ops:
+          - fn: tf.math.add
+            kwargs:
+              y: 1.
+          - fn: tf.math.subtract  # The goal here is to see the end-to-end functionality of tf_native_op without modifying the tests
+            kwargs:
+              y: 1.
```

Comment:
This will run both the python and python-jvm end-to-end tests for verification with the tf_native_op.

Reply:
is there any way to carefully verify that the model is actually using these tf functions in the jvm side?  I guess the "check that the ranking and classification scores are the same within 1e-6" test does that?

Reply:
Yes, it should. Here, I've set them to effectively cancel out and no do anything effective because I didn't want to modify all the other tests. I can do that if you think that's better.